charts/opentelemetry-demo/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.3.1
+version: 0.5.0
```

```suggestion
version: 0.4.0
```
Chart version doesn't need to align with the demo version used by default.